company-profiles/renaissance.md

```diff
@@ -0,0 +1,28 @@
+# Renaissance 
+
+## Company blurb
+
+Renaissance’s solutions help educators analyze, customize, and plan personalized learning paths for students, allowing time for what matters—creating energizing learning experiences in the classroom.
+
+## Mission
+
+To accelerate learning for all children and adults of all ability levels and ethnic and social backgrounds worldwide.
+
+## Impact
+
+More than one-third of U.S. schools use Renaissance solutions to accelerate learning for millions of students.
+
+## Products
+
+Accelerated Reader, Accelerated Math, Star Assessments, MyON, Freckle, myIDGIs for Pre-schools, Nearpod, Lalilo
+* [Schoolzilla](https://www.renaissance.com/products/schoolzilla/) - Giving district and school leaders the relevant data they need to quickly answer questions around how their district, schools, and students are performing.
+
+
+## Remote and Location Info
+
+* Headquartered in the US - Wisconsin Rapids, WI
+* [Corporate Office Locations List](https://www.renaissance.com/about-us/contact-us/#office-locations)
+
+## Open Jobs and Applying
+
+[Click Here to apply](https://careers.smartrecruiters.com/Renaissance)
```

Comment:
The layout and headings are standardised, see your other PR for correct layout #1687